usecase/src/test/java/org/mobilitydata/gtfsvalidator/usecase/ProcessParsedAgencyTest.java

```diff
@@ -0,0 +1,69 @@
+package org.mobilitydata.gtfsvalidator.usecase;
+
+import org.junit.jupiter.api.Test;
+import org.mobilitydata.gtfsvalidator.domain.entity.ParsedEntity;
+import org.mobilitydata.gtfsvalidator.domain.entity.gtfsentity.Agency;
+import org.mobilitydata.gtfsvalidator.usecase.port.GtfsDataRepository;
+import org.mobilitydata.gtfsvalidator.usecase.port.GtfsSpecRepository;
+import org.mobilitydata.gtfsvalidator.usecase.port.ValidationResultRepository;
+import org.mockito.ArgumentMatchers;
+import org.mockito.InOrder;
+import org.mockito.Mockito;
+
+import static org.mockito.Mockito.*;
+
+class ProcessParsedAgencyTest {
+
+    GtfsSpecRepository mockGtfsSpecRepo = mock(GtfsSpecRepository.class);
```

Comment:
Those mocks should not be shared between test executions.
Move this into tests that require them or into an `@beforeEach` or `@beforeAll` annotated setup method